test/Microsoft.ML.Tests/NerTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using Apache.Arrow;
+using ICSharpCode.SharpZipLib.Tar;
+using Microsoft.Data.Analysis;
+using Microsoft.ML.Data;
+using Microsoft.ML.RunTests;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.TestFramework.Attributes;
+using Microsoft.ML.TorchSharp;
+using Microsoft.ML.TorchSharp.NasBert;
+using TorchSharp;
+using Xunit;
+using Xunit.Abstractions;
+using static TorchSharp.torch.utils;
+
+namespace Microsoft.ML.Tests
+{
+    [Collection("NoParallelization")]
+    public class NerTests : TestDataPipeBase
+    {
+        public NerTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        private class TestSingleSentenceData
+        {
+            public string Sentence;
+            public string[] Label;
+        }
+
+        private class Label
+        {
+            public string Key { get; set; }
+        }
+
+        [Fact]
+        public void TestSimpleNer()
+        {
+            var labels = ML.Data.LoadFromEnumerable(
+                new[] {
+                new Label { Key = "PERSON" },
+                new Label { Key = "CITY" },
+                new Label { Key = "COUNTRY"  }
+                });
+
+            var dataView = ML.Data.LoadFromEnumerable(
+                new List<TestSingleSentenceData>(new TestSingleSentenceData[] {
+                    new TestSingleSentenceData()
+                    {   // Testing longer than 512 words.
+                        Sentence = "Alice and Bob live in the USA",
+                        Label = new string[]{"PERSON", "0", "PERSON", "0", "0", "0", "COUNTRY"}
+                    },
+                     new TestSingleSentenceData()
+                     {
+                        Sentence = "Alice and Bob live in the USA",
+                        Label = new string[]{"PERSON", "0", "PERSON", "0", "0", "0", "COUNTRY"}
```

Comment:
Is `"0"` here being used as a constant value for marking token/word that doesn't have a label? What if someone wants to label `"0"`..

Reply:
You can use anything you want that isn't in the labels IDataView (line 48) for marking a token without a label. I just used 0 cause its not in that list, but if have a zero label just make sure its in the label IDataView and then pick a different value.